core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -6407,13 +6407,29 @@ private Name() {} // prevent instantiation
       "secretKey"
   };
 
+  private static final String NAMESERVICE_PATTERN_STRING = "([a-zA-Z_\\-0-9\\.]+)";
+  private static final String ALLUXIO_MASTER_ID_PATTERN_STRING = "([a-zA-Z_\\-0-9\\.]+)";
+  private static final String ZOOKEEPER_NODE_ID_PATTERN_STRING = "([a-zA-Z_\\-0-9\\.]+)";
+
   /**
    * A set of templates to generate the names of parameterized properties given
    * different parameters. E.g., * {@code Template.MASTER_TIERED_STORE_GLOBAL_LEVEL_ALIAS.format(0)}
    */
   @ThreadSafe
   public enum Template {
     LOCALITY_TIER("alluxio.locality.%s", "alluxio\\.locality\\.(\\w+)"),
+    LOGICAL_MASTER_NAMESERVICES("alluxio.master.nameservices.%s",
```

Comment:
Please prepend `MASTER_` to all these new props.

Reply:
I'll clarify this since I originally requested the renaming:

- `LOGICAL_MASTER_NAMESERVICES` -> `MASTER_LOGICAL_NAMESERVICES`
- `LOGICAL_MASTER_RPC_ADDRESS` -> `MASTER_LOGICAL_RPC_ADDRESS`
- `LOGICAL_ZOOKEEPER_NAMESERVICES` -> `MASTER_LOGICAL_ZOOKEEPER_NAMESERVICES`
- `LOGICAL_ZOOKEEPER_ADDRESS` -> `MASTER_LOGICAL_ZOOKEEPER_ADDRESS`

Reply:
fix